layouts/page/cudf-pandas.html

```diff
@@ -90,7 +90,9 @@ <h2><i class="fa-light fa-square-info"></i> How to Use It </h2>
         <div class="col-md-6 py-3">
           <h2><i class="fa-light fa-gauge-high"></i> 150x Faster, Zero Code Change </h2>
           <canvas id="perfchart" width="500" height="375"></canvas>
-          <p class="fs-7 fw-lighter text-center">* Grace Hopper vs Intel Xeon Platinum 8480CL CPU <a
+          <p class="fs-7 fw-lighter text-center">* Standard DuckDB Data Benchmark (5GB) <br>
+            GPU: NVIDIA Grace Hopper, CPU: Intel® Xeon® Platinum 8480C <br>
+            pandas v2.2, RAPIDS cuDF v23.10 <a
```

Comment:
Is there a way to get data for a newer cuDF version? The data from pandas 2.2 (published January 19, 2024) would be newer than the corresponding cuDF data from 23.10.

Reply:
Eventually - this is the latest we have.